lighthouse-core/audits/accessibility/aria-required-parent.js

```diff
@@ -23,7 +23,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Some ARIA child roles must be contained by specific parent roles to ' +
       'properly perform their intended accessibility functions. ' +
-      '[Learn more](https://dequeuniversity.com/rules/axe/4.4/aria-required-parent).',
+      '[Learn more about ARIA roles and their required parent element](https://dequeuniversity.com/rules/axe/4.4/aria-required-parent).',
```

Comment:
```suggestion
      '[Learn more about ARIA roles and required parent element](https://dequeuniversity.com/rules/axe/4.4/aria-required-parent).',
```

(or add "their" to the string above in aria-required-children ?)